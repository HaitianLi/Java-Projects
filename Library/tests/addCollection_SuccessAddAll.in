add collection books.csv
exit