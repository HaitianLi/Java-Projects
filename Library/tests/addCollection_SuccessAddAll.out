user: 17 books successfully added.

user: Ending Library process.
